include/deal.II/base/symmetric_tensor.h

```diff
@@ -3544,6 +3544,121 @@ DEAL_II_HOST constexpr inline DEAL_II_ALWAYS_INLINE
   return result;
 }
 
+/**
+ * An enumeration to describe the desired symmetry of the
+ * symmetric fourth order tensor
```

Comment:
"the symmetric fourth order tensor" assumes that the reader is already familiar with what operation you are talking about. But there is no reference to any operation here (or to a function that uses this enum). Would you mind adding this piece of information? Perhaps in the following way:
```suggestion
 * An enumeration to describe the desired symmetry one wants to obtain in
 * symmetrizing a fourth order tensor. This enum class is used by the symmetrize()
 * function.
```